src/site/filters/liquid.js

```diff
@@ -12,10 +12,6 @@ function getPath(obj) {
 module.exports = function registerFilters() {
   const { cmsFeatureFlags } = global;
 
-  // Set timeout option to something higher (20mins)
-  // eslint-disable-next-line no-new
-  new liquid.Context({ timeout: 1200000 });
-
```

Comment:
Was this removal intentional? Seems like maybe it was lost during a merge resolution?

Reply:
Ah true

Reply:
Updated! 🙂